src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PersonRow } from '../PersonRow/PersonRow';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState<boolean>(false);
```
Consider adding generic types here to keep the consistency of your code